_gsocproposals/2022/proposal_ACTS_GPU_pipeline_optimization.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Acts GPU R&D - Optimization of GPU tracking pipeline
+layout: gsoc_proposal
+project: Acts
+year: 2022
+difficulty: high
+duration: 175
+mentor_avail: June-August
+organization:
+  - LBNL
+---
+
+## Description
+
+A Common Tracking Software ([Acts][Acts]) is a general tracking software toolkit for High Energy Physics experiments. 
+The Acts collaboration has launched several R&D lines ([vecmem][vecmem], [detray][detray], and [traccc][traccc]) 
+for GPU acceleration by parallelizing the track reconstruction. vecmem is a memory management toolkit which provides 
```

Comment:
For readability, maybe emphasize or use quotes for `vecmem`, `detray` and `traccc` when they start a sentence.